File: go/B5_28235.go
// https://www.acmicpc.net/problem/28235
package main

import (
	"fmt"
)

func main() {
	myMap := map[string]string{
		"SONGDO":    "HIGHSCHOOL",
		"CODE":      "MASTER",
		"2023":      "0611",
		"ALGORITHM": "CONTEST",
	}
	var say string
	fmt.Scan(&say)
	fmt.Println(myMap[say])
}

File: python/B5_28235.py
# https://www.acmicpc.net/problem/28235
dic = {
    "SONGDO" : "HIGHSCHOOL",
    "CODE" : "MASTER",
    "2023" : "0611",
    "ALGORITHM" : "CONTEST"
}

if __name__ == "__main__" :
    say = input()
    print(dic.get(say))
